### PR TITLE
Create UpdateAFServersUser.sql

### PR DESCRIPTION
**Pull request by derekendres**

### CODE REVIEW

**Review by kelseyduffy — APPROVED** [review]

**Comment by dcbroad3** [comment]
Does this just bypass the .gitignore rule? Is that really what we want to do? Seems like the .gitignore rule should also be amended in some way.

**Comment by dcbroad3** [comment]
> 
> 
> Does this just bypass the .gitignore rule? Is that really what we want to do? Seems like the .gitignore rule should also be amended in some way.

@derekendres pinging so this doesn't sit too long.

File: LocalArtifacts/UpdateAFServersUser.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+USE [PIFD]
```
[the remark this answers is not in the document]
Comment (cbardini) [reply LocalArtifacts/UpdateAFServersUser.sql:1]: @derekendres This PR seems to have been here a while. I agree with Doug's comment as anything under LocalArtifacts will be gitignored.

[the remark this answers is not in the document]
Comment (cbardini) [reply LocalArtifacts/UpdateAFServersUser.sql:1]: I updated the ```.gitignore``` to exclude ```UpdateAFServersUser.sql```

**Review by cbardini — APPROVED** [review]

**Pull request merged**